genienlp/ned/bootleg.py

```diff
@@ -64,10 +63,13 @@ def __init__(self, args):
 
         with open(f'{self.args.database_dir}/wiki_entity_data/type_mappings/wiki/qid2typenames.json') as fin:
             self.entityqid2typenames = ujson.load(fin)
+
+        # large print
+        self.entityqid2typenames['Q1548123'] = ['format']
```

Comment:
Uh? Why is this here? In the code?

Reply:
`qid2typenames` map qids to an unordered list of types. I noticed the type we want is sometimes not ranked highly. So I have to do some reordering. I don't want to modify the files directly cause I'd forget what I changed later. Instead do it in the code for visibility and version tracking.

Reply:
This is the wrong way around it. Changing the code is harder. We don't want hacks in the code that confuse readers and contributors. If you want to version the bootleg input files, you can do it in the workdirs (with some light-weight override file, not by tracking the full bootleg file)